src/java/org/apache/cassandra/service/accord/AccordKeyspace.java

```diff
@@ -156,12 +174,33 @@ ImmutableSortedMap<Timestamp, ByteBuffer> getValues(CommandsForKey cfk)
         }
     }
 
+    private enum TokenType
+    {
+        Murmur3((byte) 1),
```

Comment:
TODO add more partitioner support... our tests only use murmur3 so did this just for place holder